AnimalsTest/IBigCats.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AnimalsTest
```

Comment:
program.cs uses file scoped namespaces.
It is a good practice to use consistent style in all the files of the project. Analyzers may help detecting it.

Reply:
This applies to all the files